packages/fxa-auth-server/lib/senders/emails/templates/postAddRecoveryPhone/en.ftl

```diff
@@ -0,0 +1,12 @@
+postAddRecoveryPhone-subject = Backup recovery phone added
+postAddRecoveryPhone-title = You created a new backup recovery method
+# Variables:
+#  $lastFourPhoneNumber (String) - The last four digits of the user's phone number
+postAddRecoveryPhone-description = You added ••••••{ $lastFourPhoneNumber } as your backup recovery phone
```

Comment:
Sine not every country uses this format, I'm not sure we want to do that? It wouldn't match Figma anyway. 🤔

Reply:
This is a nice-to-have for readability, not a requirement, but we might need to review anyway once we support more countries because the number of digits won't be the same